src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -59,13 +62,34 @@
      * @return The internal chronicle id for a device. It can be used to track a single device across resets, app uninstalls,
      * etc.
      */
+    @Deprecated( since = "apps v2" )
     @POST( BASE + STUDY_ID_PATH + PARTICIPANT_ID_PATH + DATASOURCE_ID_PATH )
     UUID enrollSource(
             @Path( STUDY_ID ) UUID studyId,
             @Path( PARTICIPANT_ID ) String participantId,
             @Path( DATASOURCE_ID ) String datasourceId,
             @Body Optional<Datasource> datasource );
 
+    /**
+     * Apps v2: Enrolls a participant's data datasource in a study.
+     *
+     * @param organizationId Id of the organization to which study belongs
+     * @param studyId        The id of the study with which to enroll the partipant's datasource.
+     * @param participantId  The participant id which the device will be associated with.
+     * @param datasourceId   A datasource specific id.
+     * @param datasource     Datasource specific information.
+     * @return The internal chronicle id for a device. It can be used to track a single device across resets, app uninstalls,
+     * etc.
+     */
+
+    @POST( BASE + ORGANIZATION_ID_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + DATASOURCE_ID_PATH )
+    UUID enrollSourceV2(
```

Comment:
At some point in the future, the "v1" endpoints won't exist and we'll be left with `methodV2` all over the place, and I don't think that's ideal. Furthermore, the actual API doesn't distinguish between v1 and v2, i.e. this is not a thing:

```
POST /v1/{study_id}/{participant_id}/{datasource_id}
POST /v2/{org_id}/{study_id}/{participant_id}/{datasource_id}
```

... so I'm not convinced that we should do this with the method names.

The API change we're making is that we're introducing endpoints that take an additional parameter to identify the organization. For all the `V2` methods, I prefer naming them something more semantically meaningful. For example:

```
// v1
UUID enrollSource(...)

// v2
UUID enrollDataSourceInOrgStudy(..)
// or
UUID orgStudyEnrollment(...)
// or
UUID registerForOrgStudy(...)
```